plasmoid/contents/ui/main.qml

```diff
@@ -233,29 +253,45 @@ Item {
                             triggeredOnStart: false
 
                             onTriggered: {
-                                valuesLock = false
                                 executable.exec(plasmoid.configuration.executable + ` set-brightness ${bus_id} ${brightness}`)
                             }
                         }
 
-                        onMoved: () => {
-                            // Should also be locked during mouse wheel scrolling.
-                            valuesLock = true
-                            brightness = value
+                        // Backport of https://invent.kde.org/plasma/libplasma/-/commit/aea3d4b131070d8388edf84c9c2a32f7b4203617
```

Comment:
If I understood this correctly, this code doesn't need to be implemented for the KF6 version because it's built into PlasmaComponents.Slider?

Reply:
Maybe? It depends on how the distro updates KF6 versions.